frontend/src/views/AdminPages/BranchesPage/BranchComponent/BranchComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+// core components
+import Button from "components/CustomButtons/Button.js";
+
+export default function BranchComponent({branch, onEditClick, onDeleteClick}) {
```

Comment:
Don't forget to validate your props.

Reply:
Did my first attempt. Would like your feedback here.